Traceback (most recent call last):
  File "/rds/general/user/dbk24/home/code_and_analysis/hitachi_meter_reading_retrieval_off_campus_v3.py", line 83, in <module>
    df = pl.read_csv(proc.stdout, infer_schema_length=50_000)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/dbk24/home/miniforge3/envs/irpenv_1/lib/python3.11/site-packages/polars/_utils/deprecation.py", line 128, in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/dbk24/home/miniforge3/envs/irpenv_1/lib/python3.11/site-packages/polars/_utils/deprecation.py", line 128, in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/dbk24/home/miniforge3/envs/irpenv_1/lib/python3.11/site-packages/polars/_utils/deprecation.py", line 128, in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/dbk24/home/miniforge3/envs/irpenv_1/lib/python3.11/site-packages/polars/io/csv/functions.py", line 549, in read_csv
    df = _read_csv_impl(
         ^^^^^^^^^^^^^^^
  File "/rds/general/user/dbk24/home/miniforge3/envs/irpenv_1/lib/python3.11/site-packages/polars/io/csv/functions.py", line 697, in _read_csv_impl
    pydf = PyDataFrame.read_csv(
           ^^^^^^^^^^^^^^^^^^^^^
io.UnsupportedOperation: underlying stream is not seekable
